communityApp/API/Interest.js

```diff
@@ -0,0 +1,18 @@
+import { INTEREST_TYPE_FETCH_URL } from "../constants/Database";
+
+const path = 'interest_type/';
```

Comment:
what is this for?

Reply:
It is explained in the function comment, but it will be used for the signup process. I need to get the list of interest types because newbies select their interest types during the signup process and so I have to fetch the list of them to give them options.